tests/components/tplink/conftest.py

```diff
@@ -30,3 +87,52 @@ def entity_reg_fixture(hass):
 @pytest.fixture(autouse=True)
 def tplink_mock_get_source_ip(mock_get_source_ip):
     """Mock network util's async_get_source_ip."""
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock, None, None]:
+    """Override async_setup_entry."""
+    with patch("homeassistant.components.tplink.async_setup", return_value=True), patch(
+        "homeassistant.components.tplink.async_setup_entry", return_value=True
```

Comment:
Maybe this should also use patch_multiple?

Reply:
Done